libr/core/cmd_print.c

```diff
@@ -286,6 +286,7 @@ static const char *help_msg_pd[] = {
 	"pdR", "", "recursive disassemble block size bytes without analyzing functions",
 	"pds", "[?]", "disassemble summary (strings, calls, jumps, refs) (see pdsf and pdfs)",
 	"pdt", "", "disassemble the debugger traces (see atd)",
+	"pdu", "([addr])@[addr]", "disassemble until specified address",
```

Comment:
the @ is implicit, and affects all the commands, no need to document this @[addr] here